src/Layouts/LandingPage/LandingPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import Categories from '../../components/Categories/Categories';
+import Banner from "../../components/Banner/Banner";
+import Section from '../../components/Section/Section';
+import PopularCollection from '../../components/PopularCollection/PopularCollection';
+import Footer from '../../components/Footer/Footer';
+
```

Comment:
move to all components in one index.js file. and import that file instead of importing all these components